code/modules/events/vent_clog.dm

```diff
@@ -26,10 +28,11 @@
 
 /datum/round_event/vent_clog/tick()
 	if(activeFor % interval == 0)
-		var/obj/vent = pick_n_take(vents)
+		var/obj/machinery/atmospherics/components/unary/vent = pick_n_take(vents)
+		while(vent && vent.welded)
+			vent = pick_n_take(vents)

```

Comment:
No? The event goes through the list one scrubber at a time and if a scrubber is welded or removed it will do one less scrubber. This way it picks only available scrubbers and always tries to use the same number of scrubbers.